lib/still/preprocessor.ex

```diff
@@ -2,12 +2,17 @@ defmodule Still.Preprocessor do
   alias Still.SourceFile
 
   @default_preprocessors %{
-    ".slim" => [__MODULE__.Frontmatter, __MODULE__.Slime],
-    ".slime" => [__MODULE__.Frontmatter, __MODULE__.Slime],
-    ".eex" => [__MODULE__.Frontmatter, __MODULE__.EEx],
-    ".css" => [__MODULE__.EEx, __MODULE__.CSSMinify],
-    ".js" => [__MODULE__.EEx, __MODULE__.JS],
-    ".md" => [__MODULE__.Frontmatter, __MODULE__.EEx, __MODULE__.Markdown]
+    ".slim" => [__MODULE__.Frontmatter, __MODULE__.Slime, __MODULE__.OutputPath],
+    ".slime" => [__MODULE__.Frontmatter, __MODULE__.Slime, __MODULE__.OutputPath],
+    ".eex" => [__MODULE__.Frontmatter, __MODULE__.EEx, __MODULE__.OutputPath],
+    ".css" => [
+      __MODULE__.EEx,
+      __MODULE__.CSSMinify,
+      __MODULE__.OutputPath,
+      __MODULE__.URLFingerprinting
+    ],
+    ".js" => [__MODULE__.EEx, __MODULE__.JS, __MODULE__.OutputPath, __MODULE__.URLFingerprinting],
+    ".md" => [__MODULE__.Frontmatter, __MODULE__.EEx, __MODULE__.Markdown, __MODULE__.OutputPath]
```

Comment:
Can we alias all of these? `__MODULE__` everywhere is confusing

Reply:
👍